blog/2016-04-20-futhark-and-pygame.rst

```diff
@@ -0,0 +1,214 @@
+---
+title: Creating Interactive Futhark GUIs
+author: Niels G. W. Serup
+description: Using PyGame and PyOpenCL to interact with Futhark programs in simple graphical user interfaces.
+---
+
+In the previous blog post, Troels showed you how to use Futhark's
+PyOpenCL backend to generate PNG files with Python and NumPy.  There
+was also a short mention of *interactive* examples built with PyGame.
+In this post I'll go into more detail with regards to the interactive
+examples, which have been developed by several people.
```

Comment:
"with regards to" => "regarding"

Although I think I prefer an entirely different sentence.  Something like, "in this blog post we will demonstrate how a simulation or model implemented in Futhark can be given a simple visual interface".  But better.  Maybe.